### ISSUE TITLE
flaky secondary GHA rate limit

### ISSUE THREAD

**Issue by dpc** [fedimint/fedimint#6527]
https://github.com/fedimint/fedimint/actions/runs/12229656859/job/34109742517

```
No vulnerabilities were found
Warning: 4 warnings found!
Error: You have exceeded a secondary rate limit. Please wait a few minutes before you try again. If you reach out to GitHub Support for help, please include the request ID E880:34AE38:99EF36D:13091B11:67568279. - https://docs.github.com/free-pro-team@latest/rest/overview/rate-limits-for-the-rest-api#about-secondary-rate-limits

```

**Comment by dpc** [comment]
Not clear what to do about it. We might be just running too much times at the same time sometimes, hitting GH API limits.

**Comment by elsirion** [comment]
Very curious, reading the [workflow file](https://github.com/fedimint/fedimint/actions/runs/12229656859/workflow#L3) it appears to only run twice a day, hardly something that should trigger rate limits. Are we running this elsewhere too? If so maybe we can only run the dailies with reporting to GH issues and the remaining runs just complain in the test?

**Comment by elsirion** [comment]
Fixed in #6541
